<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oknoWynikow</class>
 <widget class="QMainWindow" name="oknoWynikow">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>854</width>
    <height>572</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>854</width>
    <height>572</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wyniki</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="label_nazwa">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nazwa telefonu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_opis">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>291</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string>Opis telefonu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_zdjecie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/default.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zamknij</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>oknoWynikow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
